src/expandableCalendar/infiniteAgendaList.tsx

```diff
@@ -244,7 +242,7 @@ const InfiniteAgendaList = (props: AgendaListProps) => {
       layoutProvider={layoutProvider}
       onScroll={_onScroll}
       onVisibleIndicesChanged={_onVisibleIndicesChanged}
-      scrollViewProps={{onMomentumScrollEnd: _onMomentumScrollEnd, nestedScrollEnabled: true, refreshControl}}
+      scrollViewProps={{nestedScrollEnabled: true, ...others, onMomentumScrollEnd: _onMomentumScrollEnd}}
```

Comment:
We want to allow users to change 'nestedScrollEnabled'?

Reply:
It's a good question, but it looks to me depending on the usage, so yes.

Reply:
`viewOffset` prop is used in AgendaList component. 
It shouldn't affect anything since it's not a ScrollView prop, so I guess it's ok..

Reply:
Or we can omit is from the type here

Reply:
`const InfiniteAgendaList = (props: Omit<AgendaListProps, 'viewOffset'>) => {
`

Reply:
Yes, do you want that I will add it?